autogpt_platform/backend/backend/copilot/sdk/service_test.py

```
@@ -2247,3 +2249,112 @@ def test_unknown_tool_in_blacklist_is_ignored(self):
+                was_compacted=True,
+            )
+        assert mock_refresh.await_args.kwargs["force"] is True
+
+    @pytest.mark.asyncio
+    async def test_noop_on_first_turn_or_non_user_or_disabled(self):
+        with patch(
+            "backend.copilot.graphiti.context.refresh_warm_context",
+            new_callable=AsyncMock,
+        ) as mock_refresh:
+            for kwargs in (
+                dict(has_history=False),  # first turn
```

Ruff (0.16.0)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)